test/Benchmarks/Logs/LogBenchmarks.cs

```diff
@@ -36,34 +37,39 @@ public class LogBenchmarks
     private readonly ILogger loggerWithTwoProcessors;
     private readonly ILogger loggerWithThreeProcessors;
 
+    private readonly ILoggerFactory loggerFactoryWithNoListener;
```

Comment:
We could dispose these logger factories after the benchmarks have run, either in GlobalCleanup or implement IDisposable.

Reply:
Good call, let me fix it.

Reply:
#5330